content/index.html.haml

```diff
@@ -56,6 +56,13 @@ homepage: true
   :image => {:src => expand_link('images/conferences/devops-world-2021.png'), :height => "300px"},
   :call_to_action => {:text => 'Register for DevOps World', :href => 'https://www.cloudbees.com/devops-world-2021/registration'}}
 
+-# Hacktoberfest 2021
+- slides << {:href => 'https://hacktoberfest.digitalocean.com/',
+  :title => 'Hacktoberfest 2021',
+  :intro => "We are participating to the Hacktoberfest 2021 as the Jenkins community, and we are looking for contributors and maintainers who want to join us in October!",
+  :image => {:src => expand_link('images/hacktoberfest/logo-hacktoberfest-2021-full.svg'), :height => "400px"},
```

Comment:
Minor phrasing  changes and a switch to the image that has higher contrast when compared to the background.

Also set the height to 300 pixels so that the size of the orange bar does not change when the image changes from one panel to the next
```suggestion
  :intro => "Jenkins is participating in Hacktoberfest 2021. We are seeking contributors and maintainers who want to join us to improve Jenkins in October!",
  :image => {:src => expand_link('images/hacktoberfest/hacktoberfest_2021_social.png'), :height => "300px"},
```

Looks like this with that change:

![screencapture-mark-pc2-markwaite-net-4242-2021-09-30-12_35_29-edit](https://user-images.githubusercontent.com/156685/135511804-9085adb9-4ab3-4bd8-84da-d8999f6da02e.png)